orchagent/port.cpp

```diff
@@ -33,8 +38,8 @@ sai_status_t Port::bindAclTable(sai_object_id_t& group_member_oid, sai_object_id
         group_attrs.push_back(group_attr);
 
         group_attr.id = SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_TYPE_LIST;
-        group_attr.value.objlist.count = 1;
-        group_attr.value.objlist.list = bp_list;
+        group_attr.value.s32list.count = sizeof(bp_list)/sizeof(*bp_list);
+        group_attr.value.s32list.list = bp_list;
```

Comment:
I am not sure it is necessary to add all bind point types when you creating acl table group for this port. Since the port has only one type, we should add appropriate port type to the bind point type list. Only 1 type is needed.